slackbot/plugins/jira.py

```diff
@@ -0,0 +1,280 @@
+# -*- coding: utf-8 -*-
+from errbot import BotPlugin
+from errbot import botcmd
+from itertools import chain
+import requests
+import logging
+import json
+import re
+import os
+
+from slackbot.utils.secrets import Secrets
+
+secrets = Secrets()
+
+log = logging.getLogger(name='errbot.plugins.Jira')
+
+CONFIG_TEMPLATE = {'API_URL': secrets.get('Jira', 'JIRA_URL'),
```

Comment:
should update the based config example for ~/.secrets
we should also yknow make one